transactions.go

```diff
@@ -66,13 +66,16 @@ func (b *BuxClient) SendToRecipients(ctx context.Context, recipients []*transpor
 ) (*buxmodels.Transaction, transports.ResponseError) {
 	draft, err := b.DraftToRecipients(ctx, recipients, metadata)
 	if err != nil {
+		Log.Error().Err(err).Str("transactions", "SendToRecipients").Msg(err.Error())
 		return nil, err
 	} else if draft == nil {
+		Log.Error().Err(buxerrors.ErrDraftNotFound).Str("transactions", "SendToRecipients").Msg(buxerrors.ErrDraftNotFound.Error())
```

Comment:
I think we don't need `Str("transactions", "SentToRecipients")` - it's possible to configure logger to add "stacktrace" automatically... at least I believe it is

Reply:
Will add stacktrace